paper_sections/05-computational_and_statistical_issues.Rmd

```diff
@@ -628,9 +636,21 @@ to balance penalties between nested smooth terms (this is why nested random
+To avoid this issue, we recommend both careful choice of basis and setting model degrees of freedom so that groupwise terms are either slightly less flexible than the global term or have a smaller null space. In the examples in section III, we used smoothers with an unpenalized null space for the global smoother and ones with no null space for the groupwise terms[^gsnull]. When using thin plate splines, this can be done with splines with a lower order of derivative penalized in the groupwise smoothers than the global smoothers, as lower-order "tp" splines have fewer basis functions in the null space. For example, we used `m=2` (penalizing squared second derivatives) for the global smoother, and `m=1` (penalizing squared first derivatives) for groupwise smoothers in models *GS* and *GI*. Another option is to use a lower number of basis functions (`k`) for groupwise relative to global terms.  This will reduce the maximum flexibility possible in the groupwise terms. We do caution that these are just rules of thumb. 
+
+As noted above, interpreting the shape of global terms and group-wise deviations seperately for  *GS* models fit using tensor-product group-level terms is complicated by the fact that **mgcv** will drop some basis-functions for the group-level term to prevent perfect concurvity. 
+For tensor-product smoothers, `gam.side` will generally drop $\le k$ terms from the group-level smoother, where $k$ is the number of basis functions in the global smoother.
+The total number of terms dropped will depend on the smoothers used for the global and groupwise terms, and will be dropped essentially at random from different grouping levels. 
+This means that some groups will have a different range of potential deviations from the global smoother than others.
+This has the effect of also somewhat altering the shape of the global smooth relative to what it would be based on model *G* (the average curve through all the data); this will be a larger issue when the number of basis functions in the global smooth and the number of group levels are small. 
+We have tested the effect of this issue on our simulated `bird_move` data set and did not find that it lead to substantial bias in estimating the shape of the global smoother, relative to the amount of bias inherent in any smooth estimation method[^global_note]. However, more work on when these models accurately reconstruct global effects is still needed.
```

Comment:
This needs to be more specific as to which tensor product we refer. The issue was largely confined to the `te()` version whereas the `t2()` formulation performed well. I think we could include that new figure from the discussion in #49 here and clarify the `te()/`t2()` thing and that would I feel give sufficient basis for support for this entire section.